nav2_bt_navigator/include/nav2_bt_navigator/bt_navigator.hpp

```diff
@@ -57,60 +52,36 @@ class BtNavigator : public nav2_util::LifecycleNode
    * @param state Reference to LifeCycle node state
    * @return SUCCESS or FAILURE
    */
-  nav2_util::CallbackReturn on_configure(const rclcpp_lifecycle::State & state) override;
+  virtual nav2_util::CallbackReturn on_configure(const rclcpp_lifecycle::State & state) override;
```

Comment:
Override means that its virtual, I'm pretty sure that this is going to fail linting

Reply:
Thanks. I've fixed it.